src/github-deployments.coffee

```diff
@@ -33,19 +35,20 @@ module.exports = (robot) ->
     deployTargets = []
 
   # Status
-  robot.respond /deploy status( [0-9]+)?(?: for ([\-A-z0-9]+)\/([\-A-z0-9]+))?$/i, (msg) ->
-    unless checkConfiguration(msg)
+  robot.respond /deploy status( [0-9]+)?(?: for ([\-A-z0-9]+)\/([\-A-z0-9]+))?$/i, (res) ->
+    unless checkConfiguration(res)
       return;
 
     app = process.env.HUBOT_GITHUB_REPO
-    status_id = msg.match[1]
+    channel_ids = process.env.HUBOT_GITHUB_DEPLOY_ALLOW_CHANNEL_IDS.split(",")
```

Comment:
Same as above. It's an interesting idea for a feature, but we'd want to keep it separate and add test coverage around it.

```suggestion
```

Reply:
yup this is not finished yet